### PR TITLE
Make it easier to add more chat commands

### PR DESCRIPTION
**Pull request by Panjkrc**
Changed if statements with dictionary check

## Motivation and Context

Makes it easier to add more shorts

## How Has This Been Tested?

Tested localy

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: ruqqus/chat/chat.py [hunk 1]
```diff
@@ -22,6 +22,18 @@
...
+shorts = {
+    'shrug': r"¯\\\_(ツ)_/¯",
+    'table': '(╯° □°） ╯︵ ┻━┻',
+    'lenny': '( ͡° ͜ʖ ͡°)',
+    'untable': '┬─┬ノ( º _ ºノ)',
+    'porter': '【=◈︿◈=】',
+    'notsure': '(≖_≖ )',
+    'flushed': '◉_◉',
+    'gib': '༼ つ ◕_◕ ༽つ',
+    'sus': random.choice(['ඞඞ','ඣ','යඞ'])
```
Comment (lithiumjs) [inline ruqqus/chat/chat.py:34]: this gets selected on startup and never changes again. the old version picks a random one each time it is called. maybe replace with lambdas?

**Review by lithiumjs — CHANGES_REQUESTED** [review]
see comment above

File: ruqqus/chat/chat.py (hunk 1 above)
Comment (Panjkrc) [reply ruqqus/chat/chat.py]: fixed

**Comment by Panjkrc** [comment]
depricated

**Pull request closed without merging**